src/chart/generateCategoricalChart.tsx

```diff
@@ -1659,11 +1645,7 @@ export const generateCategoricalChart = ({
     };
 
     triggerSyncEvent(data: CategoricalChartState) {
-      const { syncId } = this.props;
-
-      if (!_.isNil(syncId)) {
-        eventCenter.emit(SYNC_EVENT, syncId, this.uniqueChartId, data);
-      }
```

Comment:
Same here, why can we remove this? Can the syncId no longer be null?

Reply:
How are we making sure it is not null? Could a user not still set it to null?

Reply:
+1

Reply:
It was just a missed. Thank you for finding